### PR TITLE
Start up

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

File: openslides/core/static/js/core/base.js [hunk 1]
```diff
@@ -69,50 +68,85 @@ angular.module('OpenSlidesApp.core', [
...
-                    newConnect();
+                    Autoupdate.newConnect();
                 }, 1000);
             };
             socket.onmessage = function (event) {
                 _.forEach(Autoupdate.messageReceivers, function (receiver) {
                     receiver(event.data);
                 });
+                // The first message is done: resolve the promise.
+                // TODO: check whether the promise is already resolved.
```
Comment (normanjaeckel) [inline openslides/core/static/js/core/base.js]: Here we need a new issue.

File: openslides/core/static/js/core/base.js [hunk 2]
```diff
@@ -230,8 +264,8 @@ angular.module('OpenSlidesApp.core', [
                 var deletedElements = [];
                 var collectionString = key;
                 _.forEach(list, function(data) {
-                    // uncomment this line for debugging to log all autoupdates:
-                    // console.log("Received object: " + data.collection + ", " + data.id);
+                    // Uncomment this line for debugging to log all autoupdates:
+                    console.log("Received object: " + data.collection + ", " + data.id);
```
Comment (normanjaeckel) [inline openslides/core/static/js/core/base.js]: Please comment this line.

File: openslides/core/static/js/core/site.js [hunk 3]
```diff
@@ -77,10 +78,8 @@ angular.module('OpenSlidesApp.core.site', [
         this.$get = ['operator', function(operator) {
             return {
                 registerScope: function (scope) {
-                    var that = this;
                     this.scope = scope;
-                    this.updateMainMenu();
-                    operator.onOperatorChange(function () {that.updateMainMenu();});
+                    //this.updateMainMenu();
```
Comment (normanjaeckel) [inline openslides/core/static/js/core/site.js]: Remove this line.

File: openslides/core/static/templates/index.html [hunk 4]
```diff
@@ -175,7 +181,7 @@
     <div class="col1" ng-class="isProjectorSidebar ? 'min' : 'max'">
         <!-- dynamic views -->
         <route-loading-indicator></route-loading-indicator>
```
Comment (normanjaeckel) [inline openslides/core/static/templates/index.html]: Remove this line. The directive is already gone.

File: openslides/users/models.py [hunk 5]
```diff
@@ -196,6 +196,17 @@ def __str__(self):
         # Return result
         return name
 
+    # TODO: This is only a temporary solution. We have to skip the autoupdate only for the
```
Comment (normanjaeckel) [inline openslides/users/models.py]: Remove this lines after #2804 ist merged.

File: openslides/core/static/js/core/base.js (hunk 1 above)
Comment (FinnStutzenstein) [inline openslides/core/static/js/core/base.js]: Done.

File: openslides/core/static/js/core/base.js (hunk 2 above)
Comment (FinnStutzenstein) [inline openslides/core/static/js/core/base.js]: Done.

File: openslides/core/static/js/core/site.js (hunk 3 above)
Comment (FinnStutzenstein) [inline openslides/core/static/js/core/site.js]: Done.

File: openslides/core/static/templates/index.html (hunk 4 above)
Comment (FinnStutzenstein) [inline openslides/core/static/templates/index.html]: Done.

File: openslides/users/models.py (hunk 5 above)
Comment (FinnStutzenstein) [inline openslides/users/models.py]: Done.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel Rebased on current master.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel Please verify: goto groups permission matrix. Reload page: `TypeError: Cannot read property 'groups_id' of null` --> operator is not load yet but the state build is already ongoing and needs the operator.

**Pull request merged**